lib/debezium/types.go

```diff
@@ -28,11 +28,16 @@ const (
 type SupportedDebeziumType string
 
 const (
-	JSON    SupportedDebeziumType = "io.debezium.data.Json"
-	Enum    SupportedDebeziumType = "io.debezium.data.Enum"
-	EnumSet SupportedDebeziumType = "io.debezium.data.EnumSet"
-	UUID    SupportedDebeziumType = "io.debezium.data.Uuid"
-	LTree   SupportedDebeziumType = "io.debezium.data.Ltree"
+	JSON     SupportedDebeziumType = "io.debezium.data.Json"
+	Enum     SupportedDebeziumType = "io.debezium.data.Enum"
+	EnumSet  SupportedDebeziumType = "io.debezium.data.EnumSet"
+	UUID     SupportedDebeziumType = "io.debezium.data.Uuid"
+	LTree    SupportedDebeziumType = "io.debezium.data.Ltree"
+	Interval SupportedDebeziumType = "io.debezium.time.Interval"
+	XML      SupportedDebeziumType = "io.debezium.data.Xml"
+
+	// Bytes
+	Bit SupportedDebeziumType = "io.debezium.data.Bits"
```

Comment:
Nit can we call this `Bits`? `Bit` makes me think this is just a single bit.

Reply:
Yeah sure